### ISSUE TITLE
Truncate will error if there is not an input string

### ISSUE THREAD

**Issue by benjaminapetersen** [addyosmani/polymer-filters#15]
Found this while iterating over a long list of items.  If the input is null/undefined rather than an empty string, the input.length & input.substring(), input.lastIndexOf() calls will error.  I hacked in this quick solution @ line 12 of filter-truncate.js:

```javascript
    // may be helpful to see the input, or perhaps should just return ' '
    if(!input) {
        return input;
    }
```
Which is not elegant but suppresses errors.  I'm thinking about the best option here & will post back or do a pull request.   Rather than expecting preprocessed data, it seems reasonable that filters would handle these kinds of cases as it makes them very easy to use.

Thinking about this a little more.
Thanks!

**Comment by addyosmani** [comment]
These should be fixed as of 0cf9aa8f61ccfc3720de7a2d9d017c97c1b79205 and your other PRs against the original issue. Thanks again @benjaminapetersen!

**Comment by benjaminapetersen** [comment]
Awesome!  Glad to help, even in these small ways.
